extensions/skins/conversation_v1/player.html

```diff
@@ -53,6 +53,8 @@
             </div>
           </div>
 
+          <button ng-click="readSelectedTextAloud()">Speak</button>
```

Comment:
This is just a placeholder for the time being.  Where should the button be located? At the navbar up top?